app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -421,6 +433,108 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_loadImageRegion_clickRegion6_region6Clicked() {
+    launchForExploration(TEST_EXPLORATION_ID_5).use {
+      startPlayingExploration()
+      waitForExplorationToBeLoaded()
+      onView(withId(R.id.submit_answer_button)).check(matches(not(isClickable())))
+      // TODO(#669): Remove explicit delay - https://github.com/oppia/oppia-android/issues/1523
+      onView(isRoot()).perform(waitFor(5000))
```

Comment:
Actually I did think of this but it didn't as expected but nevermind I can give it one more try

Reply:
we can also do one thing that we make the image view visible once it is loaded that way we can use the method displayed. How does that sound?

Reply:
Yeah that seems reasonable.

Reply:
for that, we might need to expose a method into image loader that will act as a callback

Reply:
@BenHenning can you suggest something because this seems a bit tricky to make a callback for this.

Reply:
It may be easier to use dimensions in that case. Could we do something like:

```
waitForTheView(allOf(widthId(R.id...), withNonZeroDimensions()))
```

Reply:
this is done already